CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 * [#51](https://github.com/stlehmann/PyQt5-stubs/pull/51) adds `pyqtBoundSignal.signal` hinted as `str`
 
 ### Changed
+* [#102](https://github.com/stlehmann/PyQt5-stubs/pull/102) fix `pyqtSlot` `result` parameter type to `typing.Type[object]`
```

Comment:
```suggestion
* [#102](https://github.com/stlehmann/PyQt5-stubs/pull/102) fix `pyqtSlot` parameter typing and overloads
```

Are you familiar with the suggestion feature?  I can't remember if you had used it.  It gives me a button to create a commit and includes you as the author (or both of us, or something).

Reply:
I know it exists, but I couldn't find it in the 5 seconds I looked for it. I'll try to use it in the future.